example-repo/.dmno/dmno-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+projects:
```

Comment:
Note that our example repo now needs this file, since otherwise we keep scanning upwards and find the root `pnpm-workspace.yaml` file. While I hadn't planned on that, it makes sense and is more explicit.